app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2>In-Progress Tasks</h2>
+
+<table class = "all-tasks">
+
+	<tr>
+		<th>To-Do</th>
+		<th class="center">Person</th>
+		<th class="center">Delete</th>
+		<th class="center">Completed</th>
+		<th class="center">Edit</th>
+
+
+	</tr>
+
+	<% @all_tasks.each do |task| %>
+	  <tr>
+      <td>
+				<%= link_to task.title, task_path(task.id) %>
+			</td>
+			<td class="center">
+				<% if task.person_id != nil %>
+				<% num = task.person_id %>
+				<%= Person.find(num).name %>
+				<% end %>
+			</td>
```

Comment:
I guess I wonder why you needed this check.
How can a task exist without someone who wrote it?  (as you wrote in the task model:  task belongs to a person)

If this view won't work without this check, I would guess something is missing when you create the task (the person_id?)